src/components/Layout/index.jsx

```diff
@@ -53,12 +53,19 @@ export default Layout;
 
 
 Layout.propTypes = {
+  /** Nested structure that needs this wrapper **/
```

Comment:
'React content nested inside this component.'

Reply:
Updated